docs/python_support/reference.md

```diff
@@ -598,11 +598,11 @@ For a graph client can:
 
 - request status (gRPC and REST)
 - request metadata (gRPC and REST)
-- request inference (gRPC)
+- request inference (gRPC and REST)
```

Comment:
If all endpoints are supported now on both APIs maybe we could drop this explicit info? 
I mean remove `(gRPC and REST)`